x/multisig/context_test.go

```diff
@@ -0,0 +1,65 @@
+package multisig
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"testing"
+
+	"github.com/iov-one/weave"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestContext(t *testing.T) {
+	id := func(i int64) []byte {
+		bz := make([]byte, 8)
+		binary.BigEndian.PutUint64(bz, uint64(i))
+		return bz
+	}
+
+	// sig is a signature permission, not a contract ID
+	foo := id(1)
+	sig := weave.NewCondition("multisig", "usage", foo).Address()
+	// other is a permission for some "other" contract ID
+	other := MultiSigCondition(foo).Address()
+	random := weave.NewAddress(foo)
+
+	bg := context.Background()
+	cases := []struct {
+		ctx   weave.Context
+		perms []weave.Condition
+		match []weave.Address
+		not   []weave.Address
+	}{
+		{bg, nil, nil, []weave.Address{sig, other, random}},
+		{
+			withMultisig(bg, foo),
+			[]weave.Condition{MultiSigCondition(foo)},
+			[]weave.Address{sig, other},
```

Comment:
This is odd... `sig` and `other` are the clearly same despite comment and without clear reason. I would expect one test case to match sig, the other other... Please fix this up and clarify test case.